docker-serve

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash 
+
+docker run  --rm -it -p 8084:80 -v $(pwd)/build:/usr/share/nginx/html -v $(pwd)/docker-config/vhost.conf:/etc/nginx/conf.d/default.conf  nginx
```

Comment:
An oversight on my side. I meant this line.

Reply:
I am running the app in an nginx container and mounting the build folder as a volume in the document root of the nginx container. Also I added an nginx vhost configuration file, this is to enable routing of every path to the root file (index.html) to prevent 404 error for client side routes during refresh or navigation